target/linux/ramips/dts/mt7621_tplink_er605-v2.dts

```diff
@@ -97,22 +97,22 @@
 
 		port@1 {
 			status = "okay";
-			label = "eth1";
+			label = "lan2";
```

Comment:
Why start from `2`?

Reply:
This was originally suggested by @GoetzGoerisch in this comment: https://github.com/openwrt/openwrt/pull/15865#discussion_r1666382125. There is also picture of the device which are number from 1 (wan, hence wan1) to 2 (lan2) an d so on to 5 (lan5) - so there is reasoning behind this.

Reply:
On the device the ports are numbered 1 to 5.

- Port 1 is WAN
- Port 2 is WAN/LAN
- Port 3 is WAN/LAN
- Port 4 is LAN
- Port 5 is LAN

Therefore, I suggested the above labeling.